mix.exs

```diff
@@ -3,7 +3,7 @@ defmodule Blaguth.Mixfile do
 
   def project do
     [app: :blaguth,
-     version: "1.1.1",
```

Comment:
Could you not change version. I'll update it in the separate commit; please note, it will be `v1.1.2`.